### DISCUSSION TITLE
need help with menu python

### DISCUSSION THREAD

**Discussion by Afifette** [Afifette/mdrlol#5] [Q&A] [answered]
How can i make a python menu?
Like
1 - hello
2 - world
select :

**Comment by W3bParserV2** [comment, marked as answer]
### Please stop asking dumb things and just learn the basics of the python.

**First you need to define an input. Do this :**

```python
print("1 - hello\n2 - world")
choice = input("Chose an option :")
```

**Reply by Afifette** [reply]
Okay ty
